diff --git a/src/StelSplashScreen.cpp b/src/StelSplashScreen.cpp
index 2013d42..40173dc 100644
--- a/src/StelSplashScreen.cpp
+++ b/src/StelSplashScreen.cpp
@@ -21,16 +21,20 @@
 #include "StelUtils.hpp"
 
 #include <QPainter>
+#include <QColor>
 
 SplashScreen::SplashScreenWidget* SplashScreen::instance;
 
 static QPixmap makePixmap()
 {
-	QPixmap pixmap(StelFileMgr::findFile("data/splash.png"));
+	QPixmap pixmap(QSize(90,90));
+	pixmap.fill(Qt::black);
 	QPainter p(&pixmap);
+	p.setRenderHint(QPainter::Antialiasing);
+	p.setRenderHint(QPainter::HighQualityAntialiasing);
 	p.setPen(Qt::white);
-	QFontMetrics metrics(p.font());
-	p.drawText(QPointF(metrics.averageCharWidth(), 1.3*metrics.height()), StelUtils::getApplicationVersion());
+	p.setBrush(Qt::white);
+	p.drawEllipse(0,0,90,90);
 	return pixmap;
 }
 
@@ -45,11 +49,7 @@ void SplashScreen::present()
 	instance->ensureFirstPaint();
 }
 
-void SplashScreen::showMessage(QString const& message)
-{
-	Q_ASSERT(instance);
-	instance->showMessage(message, Qt::AlignLeft|Qt::AlignBottom, Qt::white);
-}
+void SplashScreen::showMessage(QString const& message){}
 
 void SplashScreen::finish(QWidget* mainWindow)
 {
